### DISCUSSION TITLE
How do you handle dagger generated files when ignored by git but linting running?

### DISCUSSION THREAD

**Discussion by sheldonhull** [dagger/dagger#8518] [General]
```
        level=error msg="Running error: can't run linter goanalysis_metalinter\nbuildssa: failed to load package dagger: could not load export data: no export data for \"dagger/devex/internal/dagger\""
```

I noticed this and while I like not including the generated files, I think I might need them so the linting can succeed. Any ideas?

**Comment by jedevc** [comment]
Heya :wave: 

So generally, the way I handle this is by running `dagger develop` as the step in my pipeline - then I have the generated code, without needing to commit it.

If you've got your go linting running inside dagger, then this is pretty easy to accomplish, you can easily write a chainable `With` function that includes your codegen, so you can then just do `directory.With(codegen)` to generate it.

Hope that helps! :tada:
